build.gradle.kts

```diff
@@ -129,14 +130,21 @@ java {
 
 tasks.named("classpathIndexCleanup") { dependsOn("compileIntegrationTestKotlin") }
 
-fun download(url: String, output: File) {
+fun download(url: String, output: File, bearerToken: String? = null) {
   if (output.exists()) {
     println("Cached $output")
     return
   }
   println("Downloading... $url")
   assert(output.parentFile.mkdirs()) { output.parentFile }
-  Files.copy(URL(url).openStream(), output.toPath())
+  val connection = URL(url).openConnection() as HttpURLConnection
+  if (bearerToken != null) {
+    connection.setRequestProperty("Accept", "application/vnd.github+json")
+    connection.setRequestProperty("Authorization", "Bearer $bearerToken")
```

Comment:
If that does not work you might want to check:
`connection.setHeader(HttpHeaders.AUTHORIZATION, "Bearer $bearerToken")`